test/index.js

```diff
@@ -109,20 +110,35 @@ describe('Consumer', function () {
 
       consumer.on('error', function (err) {
         assert.ok(err);
-        assert.equal(err.message, 'SQS delete message failed: Delete error');
```

Comment:
The removal of this assertion was a mistake; I have re-added it to the test suite.